docs/conf.py

```diff
@@ -1,5 +1,15 @@
+import os
+import sys
+
 import sphinx_rtd_theme
 
+cwd = os.getcwd()
+project_root = os.path.dirname(cwd)
+sys.path.insert(0, project_root)
+
+import cryptoconditions     # noqa
```

Comment:
Good question. I am not sure.

The [`cookiecutter`](https://github.com/audreyr/cookiecutter) project does it this way. We could also read the file like it's being done for `bigchaindb` right now.

Regarding the need to install `cryptoconditions` itself, `readthedocs` installs a project via `python setup.py install` before building the docs.

As soon as we do slightly sophisticated things like using `doctests` or [`autorun`](https://bitbucket.org/birkenfeld/sphinx-contrib/src/55b13a74d56eebdbf3b6c11b1e73d2718ea51ab7/autorun/?at=default) (to automatically generate the output), we'll need to install the project itself.

So I wonder whether the requirement to install `cryptoconditions` is a problem in this case.

But, anyhow, I am open to change it to simply parsing `version.py`.